templates/registration/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
```

Comment:
Make sure you've added a blank line at the end of the file as per the checklist requirement.